legacy-contracts/simple-lock-legacy/src/proxy_lp.rs

```diff
@@ -0,0 +1,86 @@
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+use crate::{error_messages::CANNOT_UNLOCK_YET_ERR_MSG, locked_token::LockedTokenAttributes};
+
+#[derive(TypeAbi, TopEncode, TopDecode, NestedEncode, NestedDecode, PartialEq, Debug)]
+pub struct LpProxyTokenAttributes<M: ManagedTypeApi> {
+    pub lp_token_id: TokenIdentifier<M>,
+    pub first_token_id: TokenIdentifier<M>,
+    pub first_token_locked_nonce: u64,
+    pub second_token_id: TokenIdentifier<M>,
+    pub second_token_locked_nonce: u64,
+}
+
+#[multiversx_sc::module]
+pub trait ProxyLpModule: crate::locked_token::LockedTokenModule {
+    #[payable("*")]
+    #[endpoint(removeLiquidityLockedToken)]
+    fn remove_liquidity_locked_token(
+        &self,
+        _first_token_amount_min: BigUint,
+        _second_token_amount_min: BigUint,
```

Comment:
Why are these ignored?

Reply:
We don't interact with the exchange contracts any longer. But we've kept them to not impact the microservice implementation and just keep the initial parameters of all the left endpoints.